lib/seahorn/BvOpSem2.cc

```diff
@@ -948,7 +972,7 @@ class OpSemMemManager {
 
     // XXX hard coded values
     // align of 4
-    m_ctx.addDef(bv::bvnum(0, 2, m_efac), bv::extract(2 - 1, 0, res));
+    m_ctx.addDef(bv::bvnum(0, 3, m_efac), bv::extract(3 - 1, 0, res));
```

Comment:
Why is `3` hard-coded instead of `2`? Should this not be a function of word size or a separate parameter?